rewrite-java-21/src/main/java/org/openrewrite/java/isolated/internal/Javac.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (C) 2009-2019 The Project Lombok Authors.
```

Comment:
This looked slightly out of place at first, but I think this is a heavily reduced copy of https://github.com/projectlombok/lombok/blob/v1.18.38/src/utils/lombok/javac/Javac.java

Reply:
I copied it from the project `rewrite-templating`, but you're probably right about it being reduced copy from the Lombok class.